crates/engine/tree/src/persistence.rs

```diff
@@ -30,6 +31,8 @@ pub struct Persistence<DB> {
     provider: ProviderFactory<DB>,
     /// Incoming requests to persist stuff
     incoming: Receiver<PersistenceAction>,
+    /// Handle for the static file task.
+    static_file_handle: StaticFileTaskHandle,
```

Comment:
should we stick to `Service` instead of `task`, wdyt @fgimenez ?

Reply:
I'm fine either way but we should be consistent, if we go for `task` we should update `EthService` here https://github.com/paradigmxyz/reth/blob/main/crates/ethereum/engine/src/service.rs#L46 (and containing module)

Reply:
Let's use `Service` then